src/http/api/profile.ts

```diff
@@ -400,7 +402,7 @@ export function createGetProfileFollowersHandler(
                     page = null;
```

Comment:
I know you didn't change this code - but I'm seeing the comment about not being type-safe, I think we can do an `instanceof CollectionPage` check instead, which is much safer